jest.config.js

```diff
@@ -0,0 +1,28 @@
+module.exports = {
+  moduleNameMapper: {
+    "^vue$": "vue/dist/vue.common.js"
+  },
+  moduleFileExtensions: ["js", "json", "vue"],
+  transform: {
+    ".*\\.(vue)$": "vue-jest",
+    "^.+\\.js$": "babel-jest"
+  },
+  collectCoverage: true,
+  collectCoverageFrom: [
+    "lib/**/*.js",
+    "public/js/**/*.{js,vue}",
+    "!public/js/dist/**",
+    "!**/node_modules/**"
+  ],
+  roots: ["lib", "public/js"],
+  setupFiles: ["./lib/testSetup.js"],
+  testURL: "http://localhost/",
+  husky: {
```

Comment:
The `husky` option isn't part of the jest config and needs to remain in `package.json`. This is what's causing the Vaidation Warning when you run `npm t`.